src/components/Popups/ReplyField/PopupList.tsx

```diff
@@ -1,16 +1,23 @@
-import * as React from 'react';
+import React from 'react';
+import isEmpty from 'lodash/isEmpty';
 import { FormattedMessage } from 'react-intl';
+import ItemList from './ItemList';
 import messages from '../messages';
 import PopupBase from '../PopupBase';
 import { Options, PopupReference } from '../Popper';
-
 import './PopupList.scss';
 
 export type Props = {
+    activeItemIndex?: number;
+    itemRowAs?: JSX.Element;
+    items: Record<string, unknown>[];
+    onSelect: (index: number, event: React.SyntheticEvent) => void;
```

Comment:
Is the `event` used anywhere?

Reply:
> Is the `event` used anywhere?

not now, but I think we should provide this info to the parent and parent can choose to use it or not.